tests/MagicTest.php

```diff
@@ -17,7 +17,7 @@ final class MagicTest extends TestCase
     public $retries;
     public $backoff_factor;
 
-    protected function setUp()
+    protected function setUp(): void
```

Comment:
Little outdated on how php supports typing. This is for typing these methods right?

Reply:
Correct - Because we can now define the method as returning no value (with :void) and this is part of the method definition starting in PHPUnit 8.0.0, we need to add this to our tests